ordersettler/ordersettler.go

```diff
@@ -290,9 +290,11 @@ func (r *OrderSettler) relayBatch(
 		return fmt.Errorf("calculating max batch (hash: %s) tx fee in uusdc: %w", txHash, err)
 	}
 	if maxTxFeeUUSDC.Cmp(big.NewInt(0)) <= 0 {
-		return fmt.Errorf(
-			"batch max tx fee in uusdc is <= 0 based on configured profit margin for %s. min profit margin should be lowered based on current batch size and min fee bps",
-			settlementPayoutChainID,
+		lmt.Logger(ctx).Warn(
+			"max tx fee to maintain configured profit margin when relaying settlement is less than or equal to 0. this settlement will not be relayed until it is timed out. min profit margin should be lowered based on current batch size and min fee bps to settlements can be relayed",
+			zap.String("maxTxFeeUUSDC", maxTxFeeUUSDC.String()),
+			zap.String("settlementInitiationChainID", batch.DestinationChainID()),
+			zap.String("settlementPayoutChainID", batch.SourceChainID()),
```

Comment:
I noticed this error in the dev relayer logs. It should only enter this state if the calculated min profit is greater than the order profits right?

Reply:
yeah if solvers run into this the max tx fee is negative which means that the tx fee needed to make the order profitable is either 0 (because min profit margin is equal to the min fee bps, i.e. they are not willing to pay any of their profit to gas, which means the settlement will never land), or you have a settlement with no profit and have set a min profit margin bps which means in order to maintain that profit margin you need to have a negative tx fee, which is obviously also impossible. 

previously i had this as an error since that meant that the settlement would never go through, but now I opted to change to just a warning log since the timeout will eventually let the relay through.